src/main/java/spim/io/OMETIFFHandler.java

```diff
@@ -54,7 +54,10 @@ public OMETIFFHandler(CMMCore iCore, File outDir, String xyDev,
 		deltat = iDeltaT;
 		outputDirectory = outDir;
 		this.acqRows = acqRows;
-
+		tiles = tileCount;
+		int angles = Math.floorDiv(stacks, tiles);
```

Comment:
@dscho ```Math.floorDiv()``` is only in Java SDK 8. I know our jenkins uses SDK 6. From my point of view, we need to make it SDK 6 compatible. How do you think?